Base/QTGUI/qSlicerStyle.cxx

```diff
@@ -191,17 +192,73 @@ QRect qSlicerStyle::subControlRect(ComplexControl control, const QStyleOptionCom
 
 //------------------------------------------------------------------------------
 QPalette qSlicerStyle::standardPalette()const
+{
+  QPalette palette =  standardLightPalette();
+
+  #ifdef Q_OS_WIN
+    // Qt on macOS already dynamically picks light/dark theme based on the OS setting
+    QSettings settings("HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize", QSettings::NativeFormat);
+    if (settings.value("AppsUseLightTheme") == 0)
+      {
+      QPalette palette =  standardDarkPalette();
+      }
+  #endif
```

Comment:
I'm doing something wrong here as even if dark mode is specified for apps on Windows, it is always using the standardLightPalette instead of the standardDarkPalette when using the "Slicer" style which is OS controlled.

Reply:
Probably you do not want to redefine "QPalette" in the if block.

Reply:
ah yes. Fixed.